tools/installers/install_muskit.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+rm -rf muskit.done
+
+# Install ParallelWaveGAN
+rm -rf ParallelWaveGAN
```

Comment:
Probably better to check if there is already a parallel-wavegan installed (if already installed, print a message and skip removing it, we can always suggest to remove, but prefer not to remove without asking)? It might accidentally remove useful things ahead